app/api_clients/curation_api_client.py

```diff
@@ -0,0 +1,38 @@
+import uuid
+import requests
+import json
+from datetime import date
+
+from app.models.corpus_item_model import CorpusItemModel
+from app.models.ranked_corpus_items_instance import RankedCorpusItemsInstance
+
+
+class CurationAPIClient(object):
+    @classmethod
+    async def get_ranked_corpus_items(cls, corpus_id: str, start_date: str=None, user_id=None):
+        ranked_corpus_items_id = "NEW_TAB_EN_US"
+        if not start_date:
+            start_date = date.today().strftime("%Y-%m-%d")
+
+        query = f"""
+        query Query {{
+            scheduledSurface(id: "{ranked_corpus_items_id}") {{
+                id
+                items(date: "{start_date}") {{
+                  id
+                }}
+            }}        
+        }}
+        """
+        # Sounds like we want this to go directly to corpus API.
+        # Let's talk to backend about this
+        url = 'https://client-api.getpocket.com'
+        response = requests.post(url, json={'query': query})
```

Comment:
Can we add a few identifying headers for Apollo Studio? This will let future us know what client is making the request.


Here's a usage in metaflow example https://github.com/Pocket/dl-metaflow-jobs/blob/210a14af2259ac4b26438d20b76fd93b3e53f18f/jobs/recs_api.py#L13-L16


We have some docs on this at https://github.com/Pocket/spec/tree/main/query/client-api#query-naming, but i know that isn't very discoverable, please bear with us as we look to find a better place for this.

Reply:
Done!